core/domain/classifier_domain.py

```diff
@@ -365,3 +365,71 @@ def validate(self):
                 raise utils.ValidationError(
                     'Expected answers to be a list, received %s' %
                     grouped_answers['answers'])
+
+
+class ClassifierExplorationMapping(object):
+    """Domain object for a classifier-exploration mapping model.
+
+    A Classifier-Exploration mapping is a one-to-one relation between the
+    attributes in an exploration to the classifier model it needs to utilise.
+    The mapping is from <exp_id, exp_version, state_name> to the classifier_id.
+
+    Attributes:
+        mapping_id: str. The unique id of the classifier-exploration mapping.
```

Comment:
Actually we want to query using <exp_id, exp_version, state_name>. So would not it make more sense to use composite key? Otherwise we have to use `Query()` function to make query on non-id attribute. With composite key we can use `get_by_id`.